### PR TITLE
Image and services refresh

### PR DESCRIPTION
**Pull request by sliedig**
Added new image icons and services. Includes latest releases (except for a few that are in preview)

### CODE REVIEW

**Comment by sliedig** [comment]
Hello @rkoval, have you had a chance to review this pr?

**Comment by rkoval** [comment]
ahh sorry, i completely missed the notification for this. lemme have a look now

File: src/console-services.yml [hunk 1]
```diff
@@ -1,31 +1,63 @@
...
+  url: 'https://console.aws.amazon.com/backup/home'
+
 - command: batch
-  description: 'Batch - Fully Managed Batch Processing at Any Scale'
+  description: 'Batch - Run Batch Jobs at Any Scale'
   url: 'https://console.aws.amazon.com/batch/home'
 
-- command: billing
-  description: 'Billing - Billing & Cost Management'
-  url: 'https://console.aws.amazon.com/billing/home'
```
Comment (rkoval) [inline src/console-services.yml]: why was this removed? this seems to still be a valid link

File: src/console-services.yml [hunk 2]
```diff
@@ -99,57 +175,125 @@
   description: 'EC2 - Virtual Servers in the Cloud'
   url: 'https://console.aws.amazon.com/ec2/home'
 
+- command: ecr
+  description: 'ECR - Fully-managed Docker container registry'
+  url: 'https://console.aws.amazon.com/ecr/home'
+
 - command: ecs
-  description: 'EC2 Container Service - Run and Manage Docker Containers'
+  description: 'ECS - Run and Manage Docker Containers'
```
Comment (rkoval) [inline src/console-services.yml]: we should have the expanded name in the description. i see that there are some old names that don't follow this, but we should attempt to do this going forward to provide more definition to what the search term. adding the abbreviation after the term (e.g., `Elastic Container Service (ECS)`) could be useful too if it's more well known as its abbreviation

File: src/console-services.yml [hunk 3]
```diff
@@ -99,57 +175,125 @@
...
+  description: 'EKS - Managed Kubernetes on AWS'
+  url: 'https://console.aws.amazon.com/eks/home'
+
 - command: elasticache
   description: 'ElastiCache - In-Memory Cache'
   url: 'https://console.aws.amazon.com/elasticache/home'
 
-- command: elasticbeanstalk
-  description: 'Elastic Beanstalk - Run and Manage Web Apps'
-  url: 'https://console.aws.amazon.com/elasticbeanstalk/home'
```
Comment (rkoval) [inline src/console-services.yml]: why was this removed? this is actually one of the few dashboards i still use at my new job, haha

File: src/console-services.yml [hunk 4]
```diff
@@ -99,57 +175,125 @@
...
+  description: 'Elasticsearch Service -  Run and Scale Elasticsearch Clusters'
+  url: 'https://console.aws.amazon.com/elasticsearchservice/home'
 
 - command: elastictranscoder
   description: 'Elastic Transcoder - Easy-to-Use Scalable Media Transcoding'
   url: 'https://console.aws.amazon.com/elastictranscoder/home'
 
-- command: es
-  description: 'Elasticsearch Service - Run and Scale Elasticsearch Clusters'
-  url: 'https://console.aws.amazon.com/es/home'
```
Comment (rkoval) [inline src/console-services.yml]: seems like i'll probably need to bump the major version for these renames, since i'm not sure who all is actually using which service (although i agree that this one in particular is probably better named as expanded)

File: src/console-services.yml [hunk 5]
```diff
@@ -159,22 +303,90 @@
...
-- command: mobileanalytics
-  description: 'Mobile Analytics - Collect, View and Export App Analytics'
-  url: 'https://console.aws.amazon.com/mobileanalytics/home'
+- command: mediapackage
+  description: 'MediaPackage - Deliver video to many devices using just-in-time format conversion'
+  url: 'https://console.aws.amazon.com/mediapackage/home'
 
-- command: mobilehub
-  description: 'Mobile Hub - Build, Test, and Monitor Mobile Apps'
-  url: 'https://console.aws.amazon.com/mobilehub/home'
```
Comment (rkoval) [inline src/console-services.yml]: `mobilehub` seems deprecated, but still a valid service. we should keep this for now

File: src/console-services.yml [hunk 6]
```diff
@@ -191,50 +415,98 @@
...
+  description: 'Security Hub - AWS’s security and compliance center'
+  url: 'https://console.aws.amazon.com/securityhub/home'
+
+- command: serverlessrepo
+  description: 'Serverless Application Repository - Assemble, deploy, and share serverless applications within teams or publicly'
+  url: 'https://console.aws.amazon.com/serverlessrepo/home'
+
+- command: servermigrationservice
+  description: 'Server Migration Service - Migrate on-premises servers to AWS'
+  url: 'https://console.aws.amazon.com/servermigrationservice/home'
```
Comment (rkoval) [inline src/console-services.yml]: this rename from `servermigration` --> `servermigrationservice` is not actually a valid link (it 404s for me)

File: src/console-services.yml [hunk 7]
```diff
@@ -191,50 +415,98 @@
...
+  description: 'Single Sign-On - Manage Single Sign-On (SSO) Access to AWS Accounts and Business Applications'
+  url: 'https://console.aws.amazon.com/singlesignon/home'
+
+- command: snowball
+  description: 'Snowball - Large Scale Data Transport'
+  url: 'https://console.aws.amazon.com/snowball/home'
+
 - command: sns
-  description: 'SNS - Push Notification Service'
+  description: 'Simple Notification Service - SNS Pub/Sub, Mobile Push and SMS'
```
Comment (rkoval) [inline src/console-services.yml]: this is a good expansion here 👍

File: src/console-services.yml [hunk 8]
```diff
@@ -191,50 +415,98 @@
...
+  description: 'Step Functions - Coordinate Distributed Applications'
   url: 'https://console.aws.amazon.com/states/home'
 
 - command: storagegateway
   description: 'Storage Gateway - Hybrid Storage Integration'
   url: 'https://console.aws.amazon.com/storagegateway/home'
 
-- command: swf
-  description: 'SWF - Workflow Service for Coordinating Application Components'
-  url: 'https://console.aws.amazon.com/swf/home'
```
Comment (rkoval) [inline src/console-services.yml]: why was this removed?

File: src/console-services.yml [hunk 9]
```diff
@@ -191,50 +415,98 @@
...
+- command: sumerian
+  description: 'Sumerian - Build VR, AR, and 3D applications'
+  url: 'https://console.aws.amazon.com/sumerian/home'
 
-- command: support
-  description: 'Support - Support the success and operational health of your AWS solutions'
-  url: 'https://console.aws.amazon.com/support/home'
```
Comment (rkoval) [inline src/console-services.yml]: why was this removed?

File: src/console-services.yml [hunk 10]
```diff
@@ -191,50 +415,98 @@
...
+  description: 'Systems Manager - a Central Place to View and Manage AWS Resources'
+  url: 'https://console.aws.amazon.com/systemsmanager/home'
+
+- command: transcribe
+  description: 'Transcribe - Powerful Speech Recognition'
+  url: 'https://console.aws.amazon.com/transcribe/home'
 
-- command: trustedadvisor
-  description: 'Trusted Advisor - Optimize Performance and Security'
-  url: 'https://console.aws.amazon.com/trustedadvisor/home'
```
Comment (rkoval) [inline src/console-services.yml]: same, why  was this removed?

File: src/console-services.yml [hunk 11]
```diff
@@ -191,50 +415,98 @@
...
   url: 'https://console.aws.amazon.com/vpc/home'
 
 - command: waf
-  description: 'WAF - Filter Malicious Web Traffic'
+  description: 'WAF & Shield - Protects Against DDoS Attacks and Malicious Web Traffic'
   url: 'https://console.aws.amazon.com/waf/home'
 
+- command: wellarchitectedtool
+  description: 'Well-Architected Tool - learn best practices, measure, and improve your workloads'
+  url: 'https://console.aws.amazon.com/wellarchitectedtool/home'
```
Comment (rkoval) [inline src/console-services.yml]: this link is a 404. where did yo uget this from?

File: src/console-services.yml [hunk 12]
```diff
@@ -191,50 +415,98 @@
...
+
+- command: workdocs
+  description: 'WorkDocs - Secure Enterprise Storage and Sharing Service'
+  url: 'https://console.aws.amazon.com/workdocs/home'
```
Comment (rkoval) [inline src/console-services.yml]: another 404... is it possible that these links have been removed in the 2 months that have ellapsed since the PR was posted? 😓

File: src/console-services.yml [hunk 13]
```diff
@@ -159,22 +303,90 @@
...
+  description: 'License Manager - Set rules to manage, discover, and report third-party license usage proactively'
+  url: 'https://console.aws.amazon.com/license-manager/home'
+
+- command: lightsail
+  description: 'Lightsail - Launch and Manage Virtual Private Servers'
+  url: 'https://lightsail.aws.amazon.com/ls/webapp/home'
+
+- command: macie
+  description: 'Macie - Amazon Macie classifies and secures your business-critical content.'
+  url: 'https://macie.aws.amazon.com/'
```
Comment (rkoval) [inline src/console-services.yml]: this isn't a valid URL. i'd revert this URL to the previous one

**Comment by rkoval** [comment]
@sliedig this all looks great. sorry for the delay here, but thanks for putting the time into all of this! see the comments i left.. either address or respond to them to give me reasoning for why the changes are.

**Comment by sliedig** [comment]
> @sliedig this all looks great. sorry for the delay here, but thanks for putting the time into all of this! see the comments i left.. either address or respond to them to give me reasoning for why the changes are.

I'm on it. Didn't realise there were so many issues! Thanks for the review.

File: src/console-services.yml (hunk 2 above)
Comment (sliedig) [reply src/console-services.yml]: This is the description in the Console so I just followed suit. It's is actually called Amazon Elastic Container Service, not EC2 Container Service.

File: src/console-services.yml (hunk 1 above)
Comment (sliedig) [reply src/console-services.yml]: I think I renamed this to budgets because billing doesn't actually have an icon. Will add it back in using the aws icon.

File: src/console-services.yml (hunk 3 above)
Comment (sliedig) [reply src/console-services.yml]: ooops! My bad. Putting it back. Sorry.

File: src/console-services.yml (hunk 4 above)
Comment (sliedig) [reply src/console-services.yml]: Maybe not a bad idea as all the icons have been updated to the new design.

Comment (sliedig) [reply src/console-services.yml]: What about a command "aes" instead of "es"?

File: src/console-services.yml (hunk 5 above)
Comment (sliedig) [reply src/console-services.yml]: OK. Adding back

File: src/console-services.yml (hunk 6 above)
Comment (sliedig) [reply src/console-services.yml]: Fixed

File: src/console-services.yml (hunk 8 above)
Comment (sliedig) [reply src/console-services.yml]: Again, probably because no new icon exists yet. Have used generic integration icon instead.

File: src/console-services.yml (hunk 9 above)
Comment (sliedig) [reply src/console-services.yml]: re-added

File: src/console-services.yml (hunk 10 above)
Comment (sliedig) [reply src/console-services.yml]: Re-added

File: src/console-services.yml (hunk 11 above)
Comment (sliedig) [reply src/console-services.yml]: Fixed. This is a new service offering. The URL had changed though since launch. Fixed.

File: src/console-services.yml (hunk 12 above)
Comment (sliedig) [reply src/console-services.yml]: This is a weird one. Not sure what happened here. Have replaced this with WorkLink

File: src/console-services.yml (hunk 13 above)
Comment (sliedig) [reply src/console-services.yml]: Fixed

File: src/console-services.yml (hunk 2 above)
Comment (rkoval) [reply src/console-services.yml]: k sounds good. yeah, EC2 Container Service was [its original name](https://aws.amazon.com/about-aws/whats-new/2014/11/13/introducing-amazon-ec2-container-service/). it seems to have been rebranded since then

File: src/console-services.yml (hunk 4 above)
Comment (rkoval) [reply src/console-services.yml]: how about "ess"? it doesn't seem to conflict, and seems easier to remember with the letters matching the service without the unrelated "a"

**Comment by rkoval** [comment]
@sliedig ended up using the generic blue AWS image as the default, as the newer one you added doesn't have square dimensions and looks weird/stretched when you open up the workflow (see https://github.com/rkoval/alfred-aws-console-services-workflow/commit/46c8b9584a14459ff11ba9ad654702d52ae74ac2). feel free to update though if you can get one with square dimensions

release: https://github.com/rkoval/alfred-aws-console-services-workflow/releases/tag/v2.0
